test/src/v2/mobility/stations.ts

```diff
@@ -32,49 +32,52 @@ export function stations(
     const numberOfStations = stations.length;
 
     const expects: ExpectsType = [
-      { check: 'should have status 200', expect: res.status === 200 }
+      {check: 'should have status 200', expect: res.status === 200},
     ];
 
     if (numberOfStations != 0) {
       // Station to return - returning the first
       let countPerStation = 0;
-      stations[0].vehicleTypesAvailable!.map(v => (countPerStation += v.count));
+      stations[0].vehicleTypesAvailable!.map(
+        (v) => (countPerStation += v.count),
+      );
       returnStation = {
         id: stations[0].id,
         count: countPerStation,
-        formFactor: stations[0].vehicleTypesAvailable![0].vehicleType.formFactor
+        formFactor: stations[0].vehicleTypesAvailable![0].vehicleType
+          .formFactor,
       };
       expects.push(
         {
           check: 'stations have id',
-          expect: stations.filter(s => s!.id).length === numberOfStations
+          expect: stations.filter((s) => s!.id).length === numberOfStations,
         },
         {
           check: 'should have correct id',
           expect:
             stations.filter(
-              s =>
+              (s) =>
                 s!.id.split(':')[0].length === 3 &&
                 s!.id.split(':')[1] === 'Station' &&
-                parseInt(s!.id.split(':')[2]) > 0
```

Comment:
This is the real change as the rest is prettier.

Reply:
Side note: I think it's safer to write logic like this as `foo && foo.bar` instead of `foo!.bar`, that way it won't fail if `foo` is undefined in some cases. (That is unless we want the tests to fail in those cases 🤷)